Library/Formula/saltstack.rb

```diff
@@ -12,7 +15,7 @@ class Saltstack < Formula
   head "https://github.com/saltstack/salt.git", :branch => "develop", :shallow => false
 
   depends_on :python if MacOS.version <= :snow_leopard
-  depends_on "swig" => :build
+  depends_on "swig304" => :build
```

Comment:
I ran into this trying to bump saltstack earlier. M2Crypto is broken. It can be made to work with newer swigs but requires patching and I don't think anything's been committed anywhere since M2Crypto seems undermaintained. Some context here: https://github.com/swig/swig/issues/344 and here: https://github.com/martinpaljak/M2Crypto/issues/60#issuecomment-75735489